package.json

```diff
@@ -18,29 +18,31 @@
     "test": "jest"
   },
   "peerDependencies": {
-    "@types/react": "^16.9.0 || ^17.0.0",
-    "react": "^16.9.0 || ^17.0.0",
-    "react-dom": "^16.9.0 || ^17.0.0"
+    "@types/react": "^18.0.0",
+    "react": "^18.0.0",
```

Comment:
Not a big deal at all, but would these changes be compatible with React 16 or 17? I'm guessing not, but just double-checking.

I'm not sure if anyone else actually uses this package, but it could be nice to allow older versions if it's not a burden.

Reply:
The new Root API is React 18 only unfortunately. If we leave things as they are and use `render`, React will just behave as if its running React 17.